src/OMSimulatorLib/Component.h

```diff
@@ -85,6 +85,7 @@ namespace oms
     oms_status_enu_t deleteConnectorFromTLMBus(const ComRef& busCref, const ComRef& connectorCref);
 
     virtual oms_status_enu_t exportToSSD(pugi::xml_node& node, pugi::xml_node& ssvNode) const = 0;
+    virtual oms_status_enu_t exportToSSVTemplate(pugi::xml_node& ssvNode) {return oms_status_ok;}
```

Comment:
Return "function not implemented" instead of ok

Reply:
this if fixed with https://github.com/OpenModelica/OMSimulator/pull/832/commits/39a44a5783cf0c0172f9afebd953e3e410a9fab7